### PR TITLE
Initial changes to addon-package docs

### PR DESCRIPTION
**Pull request by maryfrances01**
MTSRE-890

### CODE REVIEW

File: content/en/docs/creating-addons/top-level-operator/addon-package.md [hunk 1]
```diff
@@ -32,15 +39,36 @@ spec:
...
+we recommend you use the Managed Tenants Bundles (MTB) facilities.
+
+The following steps are an example of generating the packageImage for the reference-addon package using the MTB flow:
+
+1. In the MTB repository, create a `package` directory and add the `manifests.yaml` inside the `package` directory.
+See the following
+ [merge request](https://gitlab.cee.redhat.com/service/managed-tenants-bundles/-/tree/main/addons/reference-addon/package)
+ for an example.
+
+2. The MTB CI creates the packageImage and the Operator Lifecycle Manager (OLM) Index Image as part of the team's
```
Comment (maryfrances01) [inline content/en/docs/creating-addons/top-level-operator/addon-package.md:69]: Is the best example to describe how the MTB CI creates the packageImage and OLM Index Image?

File: content/en/docs/creating-addons/top-level-operator/addon-package.md [hunk 2]
```diff
@@ -32,15 +39,36 @@ spec:
...
+and [addons pagerDuty](https://mt-sre.github.io/docs/creating-addons/monitoring/pagerduty_integration/)
+documentation for more information.
+
+* Additionally, all the values present in `.spec.config.addonsv1`
+ can be injected into the objects within your packageImage.
+ See the
+[package operator](https://package-operator.run/docs/guides/packaging-an-application/#go-templates)
+documentation for more information.
+
+## Tenants Onboarding Steps
```
Comment (maryfrances01) [inline content/en/docs/creating-addons/top-level-operator/addon-package.md:56]: What is the connection between the ClusterPackage manifest and Tenant Onboarding? Put another way, how do we go from having a ClusterPackage manifest to onboarding steps?

**Comment by maryfrances01** [comment]
@garrettthomaskth I have a couple of questions for clarification. Thanks!

**Review by apahim — APPROVED** [review]
/lgtm

**Pull request merged**